ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -200,7 +200,8 @@ Item {
 
             StyledText {
                 objectName: "ProfileHeader_displayName"
-                Layout.maximumWidth: root.width - Style.current.xlPadding
+                Layout.maximumWidth: root.width - verificationIcons.width - contentContainer.anchors.leftMargin - contentContainer.anchors.rightMargin -
```

Comment:
It would be nice to add this component to Storybook and do some small refactoring here:
- remove dependency on store (it's used only for some small things, can be easily removed)
- a bit bigger thing - use Control instead of Item on top level - size management should be simpler then
-

Reply:
I agree, this component is used a multiple places (ProfileContextMenu, Settings,...) in different variations. I will create the task/issue from here

Reply:
Issue created: https://github.com/status-im/status-desktop/issues/14526